public/locales/en/index.json

```diff
@@ -49,17 +49,8 @@
     "button": "Create a campaign",
     "text": "Are you ready to start a campaign?"
   },
-  "what-unites-us-section": {
-    "heading": "What unites us?",
-    "productivity-title": "Productivity",
-    "respect-title": "Respect",
-    "volunteering-title": "Volunteering",
-    "productivity-content": "All volunteers spend their free time to be part of Podkrepi.bg. They initiated their participation in the Project by themselves.",
-    "respect-content": "We treat each other with respect, as well as the people we meet in our work. We listen, we cooperate, and everyone's opinion is taken into account because it matters.",
-    "volunteering-content": "We have created a donation platform for the benefit of those in need and the organizations that help them, without seeking financial or other benefits."
-  },
-  "help-podkrepi-bg-section": {
-    "want-to-help": "I want to help Podkrepi.bg",
+  "join-podkrepi-bg-section": {
+    "heading": "Why don't you join Podkrepi.bg?",
```

Comment:
For the text here and have better engageemnt - it's better to ask the question in a positive way:
"Do you want to join Podkrepi.bg?" or as an active statement:
"Come and join Podkrepi.bg"